github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1258,6 +1258,46 @@ presubmits:
         - name: kubevirtci-coveralls
           secret:
             secretName: kubevirtci-coveralls-token
+  - name: pull-kubevirt-fossa
+    cluster: ibm-prow-jobs
+    skip_branches:
+      - release-\d+\.\d+
+    always_run: false
```

Comment:
Would it make sense to later on always run it when go.mod and go.sum gets changed?

Reply:
Yes, agree. I want to make this change after we can prove that it won't block the merge to avoid friction.

Reply:
Another PR for this is definitely ok. I am not sure we need it required. Just seeing it becoming red may be good enough in this case.

Reply:
Fair point, we might just make it run always in this PR, so that we see it fail eventually.